test/HostActivationTests/project.json

```diff
@@ -3,7 +3,7 @@
   "dependencies": {
     "Microsoft.NETCore.App": {
       "type": "platform",
-      "version": "1.0.0"
+      "version": "1.0.0-*"
```

Comment:
It is a bit tricky to know this without having the background, but the way this works is:

The actual tests themselves use the same version as the "stage0" CLI that gets installed from the build.ps1.  This version needs to stay in sync with the CLI that gets installed.  Thus, this should still be `"version": "1.0.0"`.

Same goes for the `test/TestUtils/project.json` below.  That should be "1.0.0".

The "TestAssets\TestProjects" projects/apps are testing the newly built Shared Framework from within the repo.  So up above in those projects, you have it correct, those should all be "1.0.0-*".